### DISCUSSION TITLE
getting qmcpy

### DISCUSSION THREAD

**Discussion by sanjanasivakumar7** [QMCSoftware/QMCSoftware#314] [Q&A] [answered]
Hello,
I want some help in installing the package in spyder I need this for my class in simulation.
any help will do thank you.

here is the problem,
PackagesNotFoundError: The following packages are not available from current channels:

  - qmcpy

**Comment by zitterbewegung** [comment]
What is the operating system that you are using and how are you installing the package?

**Reply by sanjanasivakumar7** [reply]
windows 11 is the OS and using conda install in the anaconda prompt

**Reply by zitterbewegung** [reply]
Please try the following command in your condo environment

 pip install qmcpy

**Reply by sanjanasivakumar7** [reply]
![image](https://user-images.githubusercontent.com/60148517/223209546-146b90e4-6c4a-4dca-b517-821c279c0f9c.png)

this is the error when I try the above command.

**Reply by zitterbewegung** [reply]
Hi can you download this and tell us if this resolves the issue? https://aka.ms/vs/17/release/vs_BuildTools.exe

**Comment by zitterbewegung** [comment]
Closing. Please read error message for what to install.

**Comment by zitterbewegung** [comment, marked as answer]
Please install https://aka.ms/vs/17/release/vs_BuildTools.exe before using qmcpy on Windows.
